src/flag_gems/csrc/cstub.cpp

```diff
@@ -20,6 +20,7 @@ TORCH_LIBRARY(flag_gems, m) {
   m.def(
       "rotary_embedding(Tensor q, Tensor k, Tensor cos, Tensor sin, Tensor? position_ids=None, "
       "bool rotary_interleaved=False) -> (Tensor, Tensor)");  // q and k may be view to other size
+  m.def("bmm(Tensor A, Tensor B) -> Tensor");
```

Comment:
Please refer to torch's op schema lang for more details about relation of the schema here and the function signature of c++ function.